examples/somersaultecu.py

```diff
@@ -2445,7 +2518,210 @@ class SomersaultSID(IntEnum):
     negative_responses=NamedItemList(),
     global_negative_responses=NamedItemList(),
     import_refs=[],
-    state_charts=NamedItemList(),
+    state_charts=NamedItemList([
+        StateChart(
+            semantic="annoyed",
+            odx_id=OdxLinkId("charts.annoyed.chart", doc_frags),
+            oid=None,
+            short_name="annoyed_chart",
+            long_name=None,
+            description=None,  #Description("State chart for a day where the ECU is grumpy",
+            #      external_docs=[],
+            #      text_identifier=None),
+            states=NamedItemList([
+                State(
+                    odx_id=OdxLinkId("charts.annoyed.states.in_bed", doc_frags),
+                    oid=None,
+                    short_name="in_bed",
+                    long_name=None,
+                    description=None,
+                ),
+                State(
+                    odx_id=OdxLinkId("charts.annoyed.states.on_street", doc_frags),
+                    oid=None,
+                    short_name="on_street",
+                    long_name=None,
+                    description=None,
```

Comment:
Looking at the python code, wouldn't it be easier and more sane to commit the XML file instead?

Reply:
yeah. at least we could/should create some of these objects from XML some time in the future. That said, in about a month's time we will switch to python 3.10 and `kw-only` dataclasses so we only need to explicitly specify the non-optional attributes...